Dockerfile

```diff
@@ -43,7 +43,6 @@ USER 10000
 COPY scripts/job-cancelled /usr/local/bin
 COPY scripts/job-failure   /usr/local/bin
 COPY scripts/job-success   /usr/local/bin
```

Comment:
Could you remove the following command in this Docker file?

https://github.com/cybozu-go/github-actions-controller/blob/af39f46e3f0f19e76b89b9a26f695cc45b6f9a3a/Dockerfile#L35
```
  && rm ./bin/RunnerService.js \
```